### ISSUE TITLE
konanc fails to execute when $KONAN_HOME contains spaces

### ISSUE THREAD

**Issue by justMaku** [JetBrains/kotlin-native#616]
If path to `KONAN_HOME ` contains spaces,  konanc will fail with:

```
Error: Could not find or load main class Source.Kotlin.dist
```

Moving KONAN_HOME directory to a path without spaces makes everything work again.

Platform: macOS 10.12.6 Beta (16G8c)
Shell: zsh

**Comment by AdrianVovk** [comment]
What do you have in `$KONAN_HOME`?

**Comment by justMaku** [comment]
`/Users/maku/Projects/Open Source/Kotlin/dist`

**Comment by AdrianVovk** [comment]
@justMaku Usually the pattern in shell is to include a `\` before the space.

So `export KONAN_HOME="/Users/maku/Projects/Open\ Source/Kotlin/dist"

The `\` means that the space is part of the string and not a separator between arguments. Single quotes also do the trick in some cases. Or sometimes (which is completely possible here), the program just doesn't handle it

**Comment by justMaku** [comment]
Yeah, but I don't specify `KONAN_HOME` as a environmental variable, so when `konanc` is run, this bash method gets called:
https://github.com/JetBrains/kotlin-native/blob/master/cmd/konanc#L54

I guess there's an bug there, but my bash-foo is not sufficient enough to propose a fix.

**Comment by AdrianVovk** [comment]
@justMaku Oh I didn't realize that. Whoops. Ok then yeah it is an issue, sorry about that

**Comment by alexander-gorshenev** [comment]
@justMaku should have been fixed by #623.
